ReflectionCodingExerciseWeek12/ReflectionCodingExerciseWeek12Test/UnitTest1.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Microsoft.CSharp;
+using System;
+using System.CodeDom;
+using System.Reflection;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ReflectionCodingExerciseWeek12;
+
+namespace ReflectionCodingExerciseWeek12Test
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        /// <summary>
+        /// Tests to see if an instance of Car was constructed.
+        /// </summary>
+        [TestMethod]
+        public void CarConstructionTest()
```

Comment:
This test method does not use reflection to create an instance of the Car class. You should use Activator.CreateInstance or similar. Same for the AirPlane constructor test. (-0.2)